android/expoview/src/main/java/host/exp/exponent/ReactNativeStaticHelpers.java

```diff
@@ -126,21 +127,6 @@ public static Object getOkHttpClient(Class callingClass) {
         .cookieJar((CookieJar) cookieJar)
         .cache(sExponentNetwork.getCache());
 
-    // pass the builder through MainApplication so that detached apps can customize it
-    try {
-      Method m = Class.forName("host.exp.exponent.MainApplication").getMethod("okHttpClientBuilder", OkHttpClient.Builder.class);
```

Comment:
Is this truly unused?

Reply:
Yes, though thanks for making me double check -- there was a stub of this method in `MainApplication.java` that I've removed as well. I grepped through the rest of the codebase and found no other references to this method. The only reason it existed was because some partner using ExpoKit long ago wanted to customize the OkHttp client, but we never used it internally.

Reply:
Thanks, that historical context is useful to know